data/card_catalogue/movement.json

```diff
@@ -37,17 +43,23 @@
     "upgrade": "",
     "tier": "gold",
     "details": {
-      "rules": "Teleport the entire party anywhere",
+      "rules": {
+        "active": "Teleport the entire party anywhere.",
+        "passive": ""
+      },
       "flavour": "Endless folds in spacetime reveal themselves to the worthy",
-      "image": "assets/core/card/illustration/Gerraro_s Door.png"
+      "image": "assets/core/card/illustration/Leopold's Door.png"
```

Comment:
Re-using the Gerraro's Door image here

for reference:

![Leopold's Door](https://user-images.githubusercontent.com/8680290/128243385-ae367832-25a7-4543-a10a-09c45daad653.png)